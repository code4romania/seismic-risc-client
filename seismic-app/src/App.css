.App {
  text-align: center;
}

.App-logo {
  float: left;
  padding-right: 32px;
}

.App-menu {
  line-height: 64px;
}

.App-header {
  background-color: #fff;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #09d3ac;
}
