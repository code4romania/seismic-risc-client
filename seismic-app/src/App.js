import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route
} from "react-router-dom";
import './styles/theme.scss';
import Home from "./containers/Home";
import Blog from "./containers/Blog";

function App() {
  return (
    <Router>
      <Switch>
        <Route exact path="/">
          <Home />
        </Route>
        <Route path="/blog">
          <Blog />
        </Route>
      </Switch>
    </Router>
  );
}

export default App;
