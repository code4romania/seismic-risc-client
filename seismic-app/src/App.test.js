import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import App from "./App";

afterEach(cleanup);

describe("App component", () => {
  it("should render correctly", () => {
    const container = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
