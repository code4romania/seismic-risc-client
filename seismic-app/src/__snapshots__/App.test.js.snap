// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly 1`] = `undefined`;
